mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -78,25 +80,27 @@ def check_app_installed(self):
         """Overrides superclass."""
         # Check that the Mobly Snippet app is installed.
         if not self._adb_grep_wrapper(
-            'pm list package | grep ^package:%s$' % self.package):
+                'pm list package | grep "^package:%s[^M]\?$"' % self.package):
```

Comment:
How did you type it? I used `Ctrl+V; Ctrl+M` in cli.
But not sure how python interprets it..